### PR TITLE
split out k64f-gcc chip base target from the frdm-k64f-gcc board-specific target

### PR DESCRIPTION
**Pull request by autopulated**
The new k64f-gcc target description can be found here: https://github.com/ARMmbed/target-kinetis-k64-gcc

@bogdanm @0xc0170

### CODE REVIEW

**Comment by autopulated** [comment]
When updating to this version, you'll need to run `yotta clean`, because the path to the link script has changed, and CMake doesn't track that correctly.

**Comment by autopulated** [comment]
(note: the base target needs to be published before publishing this one after this has been merged)

File: readme.md [hunk 1]
```diff
@@ -10,6 +10,10 @@ using GCC.
 
 
 This target description derives from the generic
-[mbed-gcc](https://github.com/ARMmbed/target-mbed-gcc) target description,
-which provides most of the information about how to run the compiler.
+[k64f-gcc](https://github.com/ARMmbed/target-k64f-gcc) target description,
```
Comment (0xc0170) [inline readme.md]: should this be https://github.com/ARMmbed/target-kinetis-k64-gcc ?

Comment (autopulated) [inline readme.md]: yes!

**Comment by 0xc0170** [comment]
LGTM

**Pull request merged**